app/controllers/api/v1/campaign_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::CampaignController < ApiController
+  def index
+    @page = params['page'] ? params['page'] : 1
+    @sort_by = params['sort_by'] ? params['sort_by'] : 'position'
+    @direction = params['direction'] ? params['direction'] : 'ASC'
```

Comment:
No need to have instance variables in this case. 

I'd also favour using symbols instead of strings to access the params hash (https://github.com/bbatsov/ruby-style-guide#symbols-as-keys  , and with more details: http://stackoverflow.com/questions/8189416/why-use-symbols-as-hash-keys-in-ruby )